deploy/examples/operator.yaml

```diff
@@ -457,6 +457,11 @@ data:
   # NOTE: the value here serves as an example and needs to be
   # updated with node labels that define domains of interest
   # CSI_TOPOLOGY_DOMAIN_LABELS: "kubernetes.io/hostname,topology.kubernetes.io/zone,topology.rook.io/rack"
+
+  # To indicate the image pull policy to be applied to the csi driver
+  ROOK_CSI_IMAGE_PULL_POLICY: "IfNotPresent"
```

Comment:
I would suggest commenting this out by default in the config map. So if the user wants something else other than `IfNotPresent`, then user can un-comment this field and use appropriate pull policy type.